CODEOWNERS

```diff
@@ -8,11 +8,14 @@
 
 * @fmvilas @derberg @dalelane @smoya @char0n @asyncapi-bot-eve
 
+/bindings/anypointmq/   @mboss37 @GeraldLoeffler
+/bindings/ibmmq/        @rcoppen
+/bindings/jms/          @rcoppen @SrfHead
+/bindings/kafka/        @lbroudoux @dalelane
+/bindings/googlepubsub/ @whitlockjc
+/bindings/solace/       @damaru-inc @CameronRushton
+/bindings/pulsar/       @VisualBean
+/bindings/sns/          @dpwdec @iancooper
+/bindings/sqs/          @dpwdec @iancooper
 
-/bindings/anypointmq/       @GeraldLoeffler
-/bindings/ibmmq/            @rcoppen
-/bindings/kafka/            @lbroudoux @dalelane
-/bindings/googlepubsub/     @whitlockjc
-/bindings/solace/           @damaru-inc @CameronRushton
-/bindings/*.json            @KhudaDad414
-/bindings/pulsar/           @VisualBean
\ No newline at end of file
+/bindings/*.json        @KhudaDad414
```

Comment:
@mboss37, @rcoppen, @SrfHead, @dpwdec, @iancooper the JSON Schema files for the bindings are as of v3 being validated alongside the specification, which is why the files have been moved from the bindings repo. @derberg will at some point send you an invite to join the spec-json-schemas repository as a maintainer so you still have full ownership of the bindings 🙂